pkg/k8s/Makefile

```diff
@@ -29,6 +29,7 @@ generate:
 	  -o . \
 	  --trim-path-prefix github.com/cilium/tetragon/pkg/k8s \
 	  --plural-exceptions TracingPolicyNamespaced:TracingPoliciesNamespaced \
+	  --plural-exceptions PodInfo:PodInfo \
```

Comment:
Is this intentional? having the name singular and plural name?
I remember there was some discussion about it, so whatever you agreed with @michi-covalent makes sense to me.

Reply:
Initially running make generate was creating cilium_io.podinfoes.yaml file, and we wanted it to be named either podsinfo or having the same name for plural and singular, so I tried to replicate how TracingPolicyNamespaced CRD had been doing it.

Michi said it's ok to use same name for both, but I tried to make podinfo work, there were some issue with the makefile, which I think michi fixed, let me pull the changes he pushed and verify.

Reply:
It is fixed now.